packages/test/test-end-to-end-tests/src/test/localTestSignals.spec.ts

```diff
@@ -258,8 +258,11 @@ describeCompat("Targeted Signals", "NoCompat", (getTestObjectProvider) => {
 	let clients: SignalClient[];
 	let provider: ITestObjectProvider;
 
-	beforeEach("setup containers", async () => {
+	beforeEach("setup containers", async function () {
 		provider = getTestObjectProvider();
+		if (provider.driver.type === "odsp") {
+			this.skip();
```

Comment:
Add a comment referencing the work item or issue for the disable including a brief comment.